src/server/scripts/Commands/cs_server.cpp

```diff
@@ -132,7 +132,10 @@ class server_commandscript : public CommandScript
     // Display the 'Message of the day' for the realm
     static bool HandleServerMotdCommand(ChatHandler* handler, char const* /*args*/)
     {
-        handler->PSendSysMessage(LANG_MOTD_CURRENT, Motd::GetMotd());
+        if (handler->IsHumanReadable())
```

Comment:
But that would be exactly the same as your sample (well, bad one because motd has only one arg) just handled one level lower and transparent to command handler